main/res/values/themes.xml

```diff
@@ -54,54 +50,51 @@
         <item name="android:textColor">@color/just_white</item>
     </style>
 
-    <style name="cgeo.ActionBarStyle.Light.Inverse" parent="@style/Widget.AppCompat.Light.ActionBar.Solid.Inverse">
-        <item name="indeterminateProgressStyle">@style/cgeo.Widget.AppCompat.Base.ProgressBar.Medium</item>
-        <item name="android:indeterminateProgressStyle" tools:ignore="NewApi">@style/cgeo.Widget.AppCompat.Base.ProgressBar.Medium</item>
-    </style>
-
-    <style name="cgeo.base" parent="@style/Theme.AppCompat">
-
-       <!-- For some reason we get a different text-size here (bug in abc?), explicitly set text styles -->
-       <item name="titleTextStyle">@style/TextAppearance.AppCompat.Widget.ActionMode.Title</item>
-       <item name="subtitleTextStyle">@style/TextAppearance.AppCompat.Widget.ActionMode.Subtitle</item>
-
-       <item name="android:buttonStyle">@style/button</item>
-       <item name="android:editTextStyle">@style/edittext</item>
-       <item name="android:windowContentOverlay">@null</item>
-       <item name="actionBarStyle">@style/cgeo.ActionBarStyle</item>
-
-        <item name="colorAccent">@color/colorAccent</item>
-
+    <style name="cgeo.base" parent="@style/Theme.AppCompat.DayNight.DarkActionBar">
         <!-- own values: colors -->
-        <item name="just_color">@color/just_black</item>
-        <item name="text_color_link">@color/link</item>
-        <item name="button_color_enabled">@color/button_enabled</item>
-        <item name="button_color_disabled">@color/button_disabled</item>
-   </style>
-
-    <!-- Identical to cgeo.base aside from different parent style and bug fixes-->
-    <style name="cgeo.base.light" parent="@style/Theme.AppCompat.Light.DarkActionBar">
+        <item name="text_color">@color/colorPrimary</item>
+        <item name="text_color_grey">@color/colorPrimaryDark</item>
+        <item name="text_color_headline">@color/colorHeadline</item>
+        <item name="text_color_hint">@color/colorPrimaryHint</item>
```

Comment:
If using the DayNight theme, we should overwrite the default attributes like: 
```
<item name="colorPrimaryHint">@color/colorPrimaryHint</item>
```

Otherwise, this results in strange effects as purple buttons (with material design), as all "non-themed" use the default attributes `colorPrimary`, `colorPrimaryHint`, ...

Our "old" attributes like "?text_color" aren't needed anymore at all, as we can/should now use `@color/colorPrimary` or `?colorPrimary` directly in the layout files.

Reply:
Hm, I'm having a hard time to understand the respective purposes of the different color names. https://material.io/blog/migrate-android-material-components provides a nice overview of the colors that come with material, see the two images about halfway down. If I'm interpreting this correctly, then `colorOnPrimary` would be the text color to display the text on a background colored with `colorPrimary`, and similar for `colorOnSecondary`/ `colorSecondary` - is that understanding correct? (That would be the opposite of what I'd used in my PR so far, but no harm done yet.)

Reply:
I'm also confused by these names. I would vote for keeping the naming of the resources understandable (`@color/text_color` etc.) But overwrite the theme default attributes `colorOnPrimary` etc... (as soon as we managed to understand what they are used for ;-))

Reply:
AFAIK, `colorPrimary` and `colorSecondary` are two different kinds of accent colors... (IMHO both orange in our case)

![grafik](https://user-images.githubusercontent.com/64581222/118409337-4720b500-b68a-11eb-9689-549f9fb08a1d.png)

Reply:
Maybe like this:
- colorPrimary as background for ActionBar, colorOnPrimary for header text
- colorSecondary as accent color
- colorBackground as background for content (cache lists etc.), colorOnBackground as what text_color is today (default text for primary content)
- for other colors we are currently using (e. g. hint color, dimmed text etc.) we can use our own variables, as there are no pendants in the default color scheme
- some other colors are unused currently (colorError, colorSurface and their respective `on` text colors; colorPrimaryVariant, colorSecondaryVariant, colorOnSecondary)

Reply:
I can tell you that `colorPrimary` partly is used as an accent color as well. Here is an example of an simple material app I wrote myself:

(notice the two different accent colors - I used the raw material theme without any modifications) 

![Screenshot_20210517_145801_com fmsys settlements_copy_270x585](https://user-images.githubusercontent.com/64581222/118493104-0922a000-b721-11eb-875e-112fa4b5c8c4.jpg)